app/controllers/confirmations_controller.rb

```diff
@@ -0,0 +1,45 @@
+class ConfirmationsController < ApplicationController
+  include UserSettable
+
+  before_action lambda {
+                  self.action_to_user_finder = {
+                    show: User.method(:find_signed),
+                    create: User.method(:find_by_email)
+                  }
+                },
+                lambda {
+                  self.action_to_user_args = {
+                    show: { args: [params[:token]], kwargs: { purpose: :account_confirmation } },
+                    create: { args: [params[:email]] }
+                  }
+                }, :set_user, only: %i[show create]
```

Comment:
Why do we use lambda instead of class methods?

Reply:
@sku-wtag Bhai, I used lambdas to perform inline processing but as the functions are getting longer and hard to read fututre PRs will remove them completely and use traditional setter functions.